src/puppetlabs/dujour/version_check.clj

```diff
@@ -79,10 +79,13 @@
         {:keys [group-id artifact-id]} (get-coords product-name)
         current-version (str (or (:version request-values) (version group-id artifact-id "")))
         version-data {:version current-version}
+        agent-os (:agent-os request-values)
         request-body (-> request-values
                          (dissoc :product-name)
+                         (dissoc :agent-os)
```

Comment:
Hmm, instead of doing this extra step...what if we just assoc'd agent OS directly in the threading macro? Like:

```
...
(assoc "agent_os" (:agent-os request-values)
...
```